packages/react/src/ActionMenu/ActionMenu.features.stories.tsx

```diff
@@ -21,7 +21,7 @@ export const SingleSelection = () => {
 
   return (
     <ActionMenu onSelect={newValue => setSelectedItem(newValue)} selectionVariant="single">
-      <ActionMenu.Button>{selectedItem}</ActionMenu.Button>
+      <ActionMenu.Button>Select a GitHub Feature</ActionMenu.Button>
```

Comment:
Could we make Feature lowercase please in all of these stories?

```suggestion
      <ActionMenu.Button>Select a GitHub feature</ActionMenu.Button>
```